ferrocene/doc/qualification-plan/src/standards/iso-26262.rst

```diff
@@ -12,207 +12,259 @@ The following are sections in |iso_ref| Part 8.
      - See sub items below
+     - See sub items below
 
    * - 11.4.6.2.a
      - :doc:`Safety Manual - Qualification Scope <safety-manual:scope>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.6.2.b
      - :doc:`Qualification Report - Acceptability statement <qualification-report:statement>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 11.4.6.2.c
      - :doc:`Qualification Report - Acceptability Statement <qualification-report:statement>`
      - N/A
+     - N/A
 
    * - 11.4.6.2.d
      - :doc:`Safety Manual - Qualification Scope <safety-manual:scope>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.6.2.e
      - :doc:`Qualification Plan - Ferrocene organization <../organization>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.6.2.f
      - :doc:`Evaluation Report - rustc - Qualification method <evaluation-report:rustc/method>`
      - N/A
+     - :doc:`Evaluation Report - ferrocene-self-test - Qualification method <evaluation-report:self-test/method>`
 
    * - 11.4.6.2.g
      - :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
      - N/A
+     - N/A
 
    * - 11.4.6.2.h
      - :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
      - N/A
+     - TODO
 
    * - 11.4.7
      - N/A
      - N/A
+     - N/A
 
    * - 11.4.8.1
      - :doc:`Evaluation Report - rustc - Qualification method <evaluation-report:rustc/method>`
      - N/A
+     - :doc:`Evaluation Report - ferrocene-self-test - Qualification method <evaluation-report:self-test/method>`
 
    * - 11.4.8.2
      - :doc:`Qualification Plan - Development process <qualification-plan:development>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.8.3
      - :doc:`Qualification Plan - Development process <qualification-plan:development>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.9
      - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
      - N/A
+     - See ``rustc`` column
 
    * - 11.4.9.1
      - See items 11.4.9.2 below
      - N/A
+     - See items 11.4.9.2 below
 
    * - 11.4.9.2
      - See sub items below
      - See sub items below
+     - See sub items below
 
    * - 11.4.9.2.a
      - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
      - N/A
+     - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
 
    * - 11.4.9.2.b
      - :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
      - N/A
+     - TODO
 
    * - 11.4.9.2.c
      - :doc:`Safety Manual - rustc - Degraded environment <safety-manual:rustc/degraded-environment>`
      - N/A
+     - N/A
```

Comment:
I think we should put something here. I don't think "Degraded environment" actually fits, I think "reaction to anomalous operating conditions shall be examined" sounds more like testing to me.

Reply:
But if degraded environment was fitting for rustc it might as well fit here.